bootstrap-app/README.md

```diff
@@ -94,10 +94,13 @@ Starts the resharding process to create a new index with the given number of sha
 the data from the old index to the new index. Both indexes will be used for ingest until
 the resharding is finalized.
 
+You MUST give the elastic_name parameter to tell CMR which cluster your index is in that is going to be resharded.
+The options are: gran-elastic and elastic
```

Comment:
Might want to change these lines to reference the variables instead of the current values. Historically, CMR readme's have often gone neglected through code changes...